docs/index.asciidoc

```diff
@@ -281,6 +281,48 @@ If you need to match multiple patterns against a single field, the value can be
         }
       }
     }
+    
+To perform matches on multiple fields just use multiple entries in the `match` hash:
+
+[source,ruby]
+    filter {
+      grok {
+        match => {
+          "speed" => "Speed: %{NUMBER:speed}"
+          "duration => "Duration: %{NUMBER:duration}"
+        }
+      }
+    }
+
+However, if one pattern depends on a field created by a previous pattern, separate these into two separate grok filters,
+as it is not guaranteed that the following will work:
+
+[source,ruby]
+    filter {
+      grok {
+        match => {
+          "message" => "Hi, the rest of the message is: %{GREEDYDATA:rest}"
+          "rest => "a number %{NUMBER"number}, and a word %{WORD:word}"
+        }
+      }
+    }
+
+You should do instead:
```

Comment:
Consider dropping the non-working example.  It looks like a viable example because of the way it's formatted. Somebody moving quickly (and not reading) could grab the counter example by mistake.

Your description and good example are very clear and can stand on their own.